backend/src/apiserver/resource/resource_manager.go

```diff
@@ -283,6 +286,32 @@ func (r *ResourceManager) CreateRun(apiRun *api.Run) (*model.RunDetail, error) {
 	}
 	// Append provided parameter
 	workflow.OverrideParameters(parameters)
+	// Patch the default value to workflow spec.
+	if common.GetBoolConfigWithDefault(HasDefaultBucketEnvVar, false) {
+		patchedSlice := make([]workflowapi.Parameter, 0)
```

Comment:
could u extract this to a function for readability similar to OverrideParameters() in previous line.

Reply:
SG. Done.